packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -7,7 +7,13 @@ exports.config = {
     // WebdriverIO allows it to run your tests in arbitrary locations (e.g. locally or
     // on a remote machine).
     runner: '<%= answers.runner %>',<%
-    if(answers.hostname && answers.port) { %>
```

Comment:
this would break existing behavior if someone uses custom port or hostname

Reply:
No, line 10 moved to line 16.